src/panels/lovelace/editor/config-elements/hui-glance-card-editor.ts

```diff
@@ -92,7 +93,7 @@ export class HuiGlanceCardEditor extends LitElement
   }
 
   get _state_color(): boolean {
-    return this._config!.state_color ?? true;
+    return Boolean(this._config!.state_color) ?? true;
```

Comment:
This won't work, it will now convert `null` to a boolean, then check if it's `null` (which it never is). So now `null` defaults to `false`.